### DISCUSSION TITLE
sqlitestudiocli.exe - Trojan.Malware.300983.susgen

### DISCUSSION THREAD

**Discussion by StillNotWorking** [pawelsalawa/sqlitestudio#4504] [General]
### Details
virustotal.com the MaxSecure scanner report 'Trojan.Malware.300983.susgen' for 'sqlitestudiocli.exe'

### Steps to reproduce
Unzip file and upload single files to VirusTotal

### SQLiteStudio version
ZIP file v.3.3.3

**Comment by pawelsalawa** [comment]
It's false positive. The package was entirely build & packages on GitHub action runner (so on clean Azure VM instance) and it's hosted on GitHub. If GitHub was hacked, we would all be in much bigger trouble ;)
